src/shared/vscode/uriHandler.ts

```diff
@@ -22,23 +22,24 @@ interface HandlerWithParser<T> {
     parser?: QueryParser<T>
 }
 
+/**
+ * Note: https://code.visualstudio.com/api/references/vscode-api#window.registerUriHandler
+ * > An extension can only register a single uri handler in its entire activation lifetime.
+ */
 export class UriHandler implements vscode.UriHandler {
     public constructor() {}
 
     private handlers: Map<string, HandlerWithParser<any>> = new Map()
 
     public async handleUri(uri: vscode.Uri): Promise<void> {
-        getLogger().verbose(`UriHandler: received request on path "${uri.path}"`)
-
         const uriNoQuery = uri.with({ query: '' }).toString()
 
         if (!this.handlers.has(uri.path)) {
-            vscode.window.showErrorMessage(
-                localize('AWS.uriHandler.nohandler', 'No handler found for: {0}', uriNoQuery)
-            )
-            getLogger().verbose(`UriHandler: no valid handler found for "${uri.path}"`)
+            showViewLogsMessage(localize('AWS.uriHandler.nohandler', 'No handler for URI: {0}', uriNoQuery))
```

Comment:
Before this PR, this would print:

    2023-12-01 09:24:05 [WARN]: UriHandler: no handler for path "/connect/codecatalyst" in handlers: Map(0) {}

After this PR it would only happen for an actually-unhandled path, and in that case would look like:

```
2023-12-01 09:13:47 [WARN]: UriHandler: no found handler for path "/foo/bar" in handlers: Map(3) {
  '/clone' => {
    handler: [AsyncFunction: cloneHandler],
    parser: [Function: parseCloneParams]
  },
  '/connect/codecatalyst' => {
    handler: [AsyncFunction: connectHandler],
    parser: [Function: parseConnectParams]
  },
  '/connect/caws' => {
    handler: [AsyncFunction: connectHandler],
    parser: [Function: parseConnectParamsOld]
  }
}
```